.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+# Standard GPflow issue template
+
+## Bug case
+
+1. Describe a bug shortly - reasons and context.
+2. Write down testable minimal working example. We appreciate if you could not publish big snippets of code telling: "Houston, we have a problem". Your code must be copypast-able and easily reproducible.
+3. Sometimes an extra information is required as well:
+    * PC, macbook configuration
+    * GPflow version or SHA which you are using
```

Comment:
I know what you mean by "SHA" but other people might not, might be worth adding an explanation that you mean the git commit, maybe even the command to obtain it (`git rev-parse HEAD` or `git log -n 1`)